src/app/pages/AccountPage/Features/SendTransaction/index.tsx

```diff
@@ -85,9 +85,11 @@ export function SendTransaction(props: SendTransactionProps) {
           >
             <TextInput
               id="recipient-id"
-              suggestions={contacts.map(contact => contact.name)}
+              suggestions={contacts.map(contact => ({ label: contact.name, value: contact.address }))}
               onSuggestionSelect={event =>
-                setRecipient(contacts.find(contact => contact.name === event.suggestion)?.address || '')
+                setRecipient(
+                  contacts.find(contact => contact.address === event.suggestion?.value)?.address || '',
```

Comment:
What's a value in adding contacts with the same names? This can be harmful imo. Shouldn't we block that behavior? cc @donouwens

Reply:
Anyway, if  we will need to change behavior I will create ClickUp task. For now I will merge this and make a release just in case we will have a deploy. Thanks Luka!